Pod/Classes/PINRemoteImageCategoryManager.m

```diff
@@ -172,7 +172,7 @@ + (void)setImageOnView:(id <PINRemoteImageCategory>)view
     if (placeholderImage) {
         [view pin_setPlaceholderWithImage:placeholderImage];
     }
-    PINRemoteImageManagerDownloadOptions options = PINRemoteImageManagerDownloadOptionsNone;
+    PINRemoteImageManagerDownloadOptions options = [view pin_defaultOptions];
```

Comment:
Since this will break any existing subclasses that don't implement this, can you make it optional in the protocol and check that the view responds to selector?